### PR TITLE
track keywords functions added

### PR DESCRIPTION
**Pull request by TheNaman047**
This PR contains logic for fetching and adding tracked keywords

### CODE REVIEW

File: functions/src/users/index.ts [hunk 1]
```diff
@@ -1,104 +0,0 @@
-import * as functions from 'firebase-functions';
-import { addTeamMember, createTeam } from '../teams';
```
Comment (jeffdh5) [inline functions/src/users/index.ts:2]: keep this file

File: functions/src/teams/index.ts [hunk 2]
```diff
@@ -1,114 +0,0 @@
-import * as functions from 'firebase-functions';
```
Comment (jeffdh5) [inline functions/src/teams/index.ts:1]: keep teams feature

File: functions/src/rankings/index.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+/* import * as functions from 'firebase-functions';
+import { dbPool } from '../helpers/pg';
+
+export const searchRankData = functions.https.onCall(async (data, context) => {
```
Comment (jeffdh5) [inline functions/src/rankings/index.ts:4]: uncomment?

File: functions/src/config.ts [hunk 4]
```diff
@@ -1,35 +1,4 @@
...
-// Set environment variables
-import * as functions from 'firebase-functions';
 
-// Initialize Postmark
-import { ServerClient } from 'postmark';
-// Initialize Stripe
-import Stripe from 'stripe';
 admin.initializeApp();
-
-// Export Storage and Firestore database and add custom settings
```
Comment (jeffdh5) [inline functions/src/config.ts:12]: can we keep the boilerplate for teams/postmark/stripe, but just turn the feature off?

Comment (jeffdh5) [reply functions/src/config.ts:12]: in a future task, we will turn all those things on.

File: functions/tsconfig.json [hunk 5]
```diff
@@ -7,7 +7,8 @@
     "sourceMap": true,
     "strict": false,
     "target": "es2017",
-    "typeRoots": ["node_modules/@types"]
+    "typeRoots": ["node_modules/@types"],
+    "skipLibCheck": true
```
Comment (jeffdh5) [inline functions/tsconfig.json:11]: why do we turn this on?

Comment (jeffdh5) [reply functions/tsconfig.json:11]: we should add typing if possible

File: functions/src/track_keywords/index.ts [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+);
+
+export const addTrackKeyword = functions.https.onCall(async (data, context) => {
+  const { uid } = context.auth;
+  const { keyword, country } = data;
+
+  try {
+    const insertRes = await dbPool.query(
+      `INSERT INTO l_trackedkeywords (keyword, country) VALUES ($1, $2)
+				  ON CONFLICT (user_id, keyword) DO UPDATE SET keyword = $2`,
```
Comment (jeffdh5) [inline functions/src/track_keywords/index.ts:26]: what is the PK? i think we don't need the user_id on this table

File: functions/src/users/index.ts (hunk 1 above)
Comment (TheNaman047) [reply functions/src/users/index.ts:2]: As discussed will add this later.

File: functions/src/teams/index.ts (hunk 2 above)
Comment (TheNaman047) [reply functions/src/teams/index.ts:1]: As discussed will add this later.

File: functions/src/rankings/index.ts (hunk 3 above)
Comment (TheNaman047) [reply functions/src/rankings/index.ts:4]: Working on it. Will send a different PR for this later.

File: functions/src/config.ts (hunk 4 above)
Comment (TheNaman047) [reply functions/src/config.ts:12]: As discussed will add this later.

File: functions/tsconfig.json (hunk 5 above)
Comment (TheNaman047) [reply functions/tsconfig.json:11]: There is a version conflict in the internal node module when I updated the firebase functions to latest version. Though I tested and it is not a logical issue but just a type issue which is preventing build of the code. Thus have added this.

File: functions/src/track_keywords/index.ts (hunk 6 above)
Comment (TheNaman047) [reply functions/src/track_keywords/index.ts]: Added by mistake, have removed it.

**Pull request merged**